client/src/slices/activeDraftSlice.ts

```diff
@@ -0,0 +1,18 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { Maybe, MongoID } from '../types';
+
+export type ActiveDraftState = Maybe<MongoID>;
+
+const initialState: ActiveDraftState = null;
+
+export const activeDraftSlice = createSlice({
+  name: 'activeDraft',
+  initialState,
+  reducers: {
+    updateDraftId: (_state, { payload }) => payload,
+  },
+});
+
+export const { updateDraftId } = activeDraftSlice.actions;
```

Comment:
So `createSlice` creates attributes on `actions` that have names indicated by the name field?

Reply:
Hmm, not quite. I think I'd say that

> `createSlice` creates an `actions` property on the slice, which takes any reducers and auto-generates action creators for them.